src/main/scala/io/viash/ViashNamespace.scala

```diff
@@ -303,12 +304,12 @@ object ViashNamespace {
         configData
     }
 
-    for (data <- if (parallel) collectedData.par else collectedData) {
+    for (data <- if (parallel) collectedData.par.iterator else collectedData) {
       // remove trailing + or ; mode character
       val commandNoMode = command.replaceFirst(""" \\?[;+]$""", "")
       val replacedCommand = 
         fields.foldRight(commandNoMode){ (field, command) => 
-          command.replaceAllLiterally(s"{$field}", data.getField(field).get)
+          command.replace(s"{$field}", data.getField(field).get)
```

Comment:
Will this still replace all occurrences of `{field}`?

Reply:
```